src/components/templates/appLayout.tsx

```diff
@@ -52,7 +51,7 @@ export const AppLayout = ({ className, topbarVariant }: { className?: string; to
 			<Sidebar />
 
 			<div className="mb-2 flex flex-1 flex-col">
-				{topbarVariant ? <ProjectConfigTopbar variant={topbarVariant} /> : null}
+				{!hideTopbar ? <ProjectConfigTopbar /> : null}
```

Comment:
Let's make the condition positive (without `!`):
```suggestion
				{hideTopbar ? null : <ProjectConfigTopbar />}
```

Reply:
changed